react/features/base/lib-jitsi-meet/native/polyfills-browser.js

```diff
@@ -259,7 +259,8 @@ function _visitNode(node, callback) {
     // location
     if (typeof global.location === 'undefined') {
         global.location = {
-            href: ''
+            href: '',
+            search: ''
```

Comment:
I'll add a "Required by" comment in order to maybe ease maintenance in the future.